project_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+proofreader src/
```

Comment:
The configuration is in `./.php_cs` like for other projects?
The other builds also targeted only `src/`, but I'll make it support multiple directories to allow multiple arguments here.